src/com/rcv/Main.java

```diff
@@ -139,18 +135,17 @@ static String executeTabulation(ElectionConfig config) {
         Tabulator tabulator = new Tabulator(castVoteRecords, config);
         // do the tabulation
         tabulator.tabulate();
-
         // generate visualizer spreadsheet data
         tabulator.generateSummarySpreadsheet(timestampString);
         // generate audit data
         tabulator.doAudit(castVoteRecords);
       } catch (Exception exception) {
-        response = String.format("ERROR during tabulation: %s", exception.toString());
-        Logger.log(response);
+        encounteredError = true;
+        Logger.severe("ERROR during tabulation: %s", exception.toString());
       }
     }
-
-    Logger.removeLogger(auditLogPath);
+    Logger.info("Done logging tabulation to %s", auditLogPath);
```

Comment:
Maybe instead consider getting rid of the above message, and instead putting one inside `removeTabulationFileLogging()` at the end that indicates that it's being removed and/or was successfully removed.

Reply:
I put it here as there is enough context to indicate the auditLogPath, and we only expect to have a single call-point so we won't need to duplicate the line.